change/office-ui-fabric-react-2019-08-07-16-29-27-spinButtonLabelFix.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Spinbutton: removed additional top and bottom margin styling overriding Label's styling and causes alignment inconsistencies with textbox, combobox, etc.",
```

Comment:
I suggest making this description more clear and concise since it'll be in the release notes.